src/org/apache/hadoop/tools/Curl.php

```diff
@@ -13,6 +13,17 @@ public function __construct($debug = false)
         $this->debug = $debug;
     }
 
+    public function cleanLastRequest()
+    {
+        unset($this->lastRequestInfoResult);
+        unset($this->lastRequestContentResult);
+        if (function_exists('gc_collect_cycles')) {
```

Comment:
I don't think this guard is necessary. The library specifies a minimum of php 5.4, and `gc_collect_cycles` has been in php since 5.3.